### ISSUE TITLE
How to get my output?

### ISSUE THREAD

**Issue by RaylinLee** [bigmb/Unet-Segmentation-Pytorch-Nest-of-Unets#65]
Hello, I followed your instructions and inputted the data I want to process into the program, but for some reason, I am not getting the desired output. I would like to ask how to solve this issue.
Best regard
Ray

**Comment by bigmb** [comment]
Try increasing the size of epochs and also check the mask results after each epoch of training and testing images. You might get a better idea about the results.
